src/components/BSelect/BSelect.test.js

```diff
@@ -0,0 +1,79 @@
+import { shallow, mount } from 'vue-test-utils'
+import BSelect from './BSelect'
+
+describe('BSelect', () => {
+  it('creates options when passed', () => {
```

Comment:
We probable want at least one snapshot test in here, to verify default looks how we expect and then when props are passed that _that_ looks how we expect.

Reply:
agreed on this